google/cloud/security/common/data_access/forseti.py

```diff
@@ -0,0 +1,199 @@
+
+        id = Column(BigInteger(), primary_key=True)
+        org_id = Column(BigInteger())
+        iam_policy = Column(Text)
+
+        def __repr__(self):
+            """String representation."""
+            return """<Policy(id='{}', type='{}', name='{}'>""".format(
+                self.id, "organization", self.org_id)
+
+        def get_resource_reference(self):
+            """Return a reference to the resource in the form (type, id)"""
+            return 'organization', self.org_id
+
+        def get_policy(self):
+            """Return the corresponding IAM policy."""
+            return self.iam_policy
+
+    class Bucket(BASE):
+        """Represents a GCS bucket item."""
+        __tablename__ = 'buckets_%s' % timestamp
+
+        id = Column(BigInteger(), primary_key=True)
+        project_number = Column(BigInteger())
+        bucket_id = Column(String(255))
+        bucket_name = Column(String(255))
+        bucket_kind = Column(String(255))
+        bucket_storage_class = Column(String(255))
+        bucket_location = Column(String(255))
+        bucket_create_time = Column(Date)
+        bucket_update_time = Column(Date)
+        bucket_selflink = Column(String(255))
+        bucket_lifecycle_raw = Column(Text)
+        raw_bucket = Column(Text)
+
+        def __repr__(self):
+            """String representation."""
+            return """<Bucket(id='{}', name='{}', location='{}')>""".format(
+                self.bucket_id, self.bucket_name, self.bucket_location)
+
+    class Organization(BASE):
+        """Represents a GCP organization."""
+        __tablename__ = 'organizations_%s' % timestamp
+
+        org_id = Column(BigInteger(), primary_key=True)
+        name = Column(String(255))
+        display_name = Column(String(255))
+        lifecycle_state = Column(String(255))
+        raw_org = Column(Text)
+        creation_time = Column(Date)
+
+        def __repr__(self):
+            """String representation."""
+            return "<Organization(id='{}', name='{}', display_name='{}')>".\
+                format(self.org_id, self.name, self.display_name)
+
+    result = (Organization,
+              [('projects', Project), ('buckets', Bucket)],
+              [OrganizationPolicy, ProjectPolicy])
+    TABLE_CACHE[timestamp] = result
+    return result
+
+
+class Importer(object):
+    """Forseti data importer to iterate the inventory and policies."""
+    DEFAULT_CONNECT_STRING = 'mysql://root@127.0.0.1:3306/forseti_security'
+
+    def __init__(self, db_connect_string=DEFAULT_CONNECT_STRING):
+        engine = create_engine(db_connect_string, pool_recycle=3600)
+        BASE.metadata.create_all(engine)
+        session = sessionmaker(bind=engine)
+        self.session = session()
+        self._get_latest_snapshot()
+
+    def _get_latest_snapshot(self):
+        """Find the latest snapshot from the database."""
+        self.snapshot = self.session.query(Snapshot).\
```

Comment:
nit: Style requires [parens](http://google.github.io/styleguide/pyguide.html?showone=Line_length#Line_length) to break a statement across multiple lines.

See here for details. One option might be break up the operations. Please fix throughout.